blackjack/blackjack_test.go

```diff
@@ -33,23 +34,37 @@ func TestNewBlackjackGame(t *testing.T) {
 		Cards: stack,
 	}
 
+	setupOutput := &bytes.Buffer{}
+	setupInput := strings.NewReader("1\nPlanty\na\n3")
+
 	g, err := blackjack.NewBlackjackGame(
 		blackjack.WithCustomDeck(deck),
+		blackjack.WithOutput(setupOutput),
+		blackjack.WithInput(setupInput),
 	)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	g.Start()
-
-	got := blackjack.ReportMap[g.Player.HandOutcome]
+	blackjack.RunCLI(g)
 
-	want := "***** Blackjack!  Player wins *****"
+	want := 3
+	got := g.Players[0].Win + g.Players[0].Lose + g.Players[0].Tie
 
 	if want != got {
-		t.Fatalf("wanted: %s, got: %s", want, got)
+		t.Fatalf("want: %d, got:%d", want, got)
 	}
 
+	outputReport := &bytes.Buffer{}
+
+	g.Players[0].GetPlayerReport(outputReport)
```

Comment:
Yes, why don't you make `GetPlayerReport` return a struct (`PlayerReport`?), and write a `String` method for it which produces this string. Then the test can verify the results without actually having to string-match.